internal/sync/sync.go

```diff
@@ -7,6 +7,7 @@ import (
 	"math"
 	"math/big"
 	"strconv"
+	sync2 "sync"
```

Comment:
We could probably do better and allow this package from the standard library to take precedence and then rename the other `dbsync` but this pattern seems to be used throughout so I have no strong opinions about it.